test/integration/good/code-gen/complex_numbers/basic_ops_mix.stan

```diff
@@ -0,0 +1,136 @@
+data {
+  int N;
+  complex_matrix[N,N] cmat;
+  complex_vector[N] cvec;
+  complex_row_vector[N] crowvec;
+  complex z;
+
+  matrix[N,N] mat;
+  vector[N] vec;
+  row_vector[N] rowvec;
+  real r;
+}
+
+parameters {
+  complex_matrix[N,N] cvmat;
+  complex_vector[N] cvvec;
+  complex_row_vector[N] cvrowvec;
+  complex zv;
+
+  matrix[N,N] vmat;
+  vector[N] vvec;
+  row_vector[N] vrowvec;
+  real v;
+}
+
+
+transformed parameters {
+  complex_matrix[N,N] gq_c_matrix;
+
+  // matrix-matrix multiply and elt
+  gq_c_matrix = cmat * cvmat;
```

Comment:
Is there a reason these are labeled gq?

Reply:
I copied/pasted and didn't change the names. I'll do it now